### DISCUSSION TITLE
Welcome to r-project-sprint-2023 Discussions!

### DISCUSSION THREAD

**Discussion by hturner** [r-devel/r-project-sprint#1] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to share and discuss potential projects to work on at [R Project Sprint 2023](https://contributor.r-project.org/r-project-sprint-2023).

These discussions are open to anyone with an interest in the development of base R, however note that the sprint activities will ultimately be driven by the sprint participants.

Comments in this space are covered by the [R Project Sprint 2023 code of conduct](https://contributor.r-project.org/r-project-sprint-2023) as well as the [GitHub community guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines). 

To create a welcoming and supportive workspace, we ask you to:

* Be welcoming, friendly and patient.
* Avoid sarcasm or belittling language.
* Respect people’s technology choices.
* Provide direct links vs “RTFM”/”Just Google it” responses.
* Use gender inclusive language (see e.g. this [factsheet on gender inclusive language](https://services.anu.edu.au/human-resources/respect-inclusion/gender-inclusive-language)).

To get started, comment below with an introduction of yourself and tell us about what you do in the R community.

**Comment by ahl27** [comment]
Hi everyone! My name is Aidan Lakshman, and I'm a PhD candidate in Biomedical Informatics at University of Pittsburgh. My doctoral work focuses on large scale comparative phylogenetic analyses. 

I develop the [SynExtend package](http://bioconductor.org/packages/release/bioc/html/SynExtend.html) on Bioconductor, and lately I've also been submitting code to the [Biostrings package](http://bioconductor.org/packages/release/bioc/html/Biostrings.html). 

I'm hoping to be able to finish my refactor of the `dendrapply` function in this project sprint! There's a link to the project [here](https://contributor.r-project.org/r-project-sprint-2023/projects/dendrapply-refactor/) if you're interested in helping out, and you can check out other stuff I work on at [my website](https://www.ahl27.com).

**Comment by itsdebartha** [comment]
Hello...
I am Debartha Paul, and I am a Master's student in Statistics at Presidency University, Kolkata. My use case for R is mainly as a student and am curious to work on it in this Project Sprint.

**Comment by mmaechler** [comment]
Hello everyone, 
I'm Martin Maechler (spelled "Mächler" locally) and have been an all time R core member.
I've been working permanently at [ETH Zurich](https://https://ethz.ch/en.html) most of my life, as Senior
Scientist and adjunct professor, and enjoyed living close by in Zurich, Switzerland. 
Happily married, father of two "boys", currently grand daddy of two, more to come, ;-)

Apart from R itself, I'm maintaining/co-authoring several CRAN packages, notably
[Matrix](https://cran.R-project.org/package=Matrix), [cluster](https://cran.R-project.org/package=cluster), [robustbase](https://cran.R-project.org/package=robustbase), [copula](https://cran.R-project.org/package=copula), [Rmpfr](https://cran.R-project.org/package=Rmpfr), [DPQ](https://cran.R-project.org/package=DPQ), [sfsmisc](https://cran.R-project.org/package=sfsmisc), [Bessel](https://cran.R-project.org/package=Bessel), [round](https://cran.R-project.org/package=round), and a couple more.
I'm co-author in other projects, notably also the packages [lme4](https://cran.R-project.org/package=lme4), [glmmTMB](https://cran.R-project.org/package=glmmTMB), [pcalg](https://cran.R-project.org/package=pcalg), [gmp](https://cran.R-project.org/package=gmp), and most recently [S7](https://cran.R-project.org/package=S7).

Within the R Core team, I've worked on numerical accuracy issues, and statistical and numerical "robustness",
regression testing (originally), reproducibility/checking of examples and other documentation, traditional graphics enhancements, and quite a few other on- and off- topics.
I've also been relatively active in the R-core--maintenance of Recommended package [nlme](https://cran.R-project.org/package=nlme).
Recently, I've also been envolved in the [OOP working group](https://rconsortium.github.io/OOP-WG/).

In teaching R, I have emphasized a human-centered programming style, emphasizing DRY as opposed WET (and defying _style guides_ which tend to mechanistic instead of creative thinking).  See e.g.
[Good Practices in R Programming](https://stat.ethz.ch/Teaching/maechler/R/useR_2014/Maechler-2014-4a4.pdf), [What I find important when R Programming](https://stat.ethz.ch/~maechler/U/R/Zurich_R-2018_ProgR-ALTREP.html), or [Helping R to be (even more) Accurate](https://stat.ethz.ch/Teaching/maechler/R/useR_2018/Maechler_Accuracy.pdf) .   .... and yes, there is much more to life than "just" R!

**Comment by dickoa** [comment]
Hi everyone,

I'm Ahmadou Dicko, I live in Dakar Senegal. I'm a statistician and a humanitarian using R since 2010. I'm the author of [robotoolbox](https://cran.r-project.org/package=robotoolbox) on CRAN, and built several other packages not submitted, currently sitting in my Gitlab repos. I'm also an [Emacs ESS](https://ess.r-project.org/) user, and I also hope to be able to contribute to it actively one day.

This Sprint is a great opportunity to learn how to contribute to R. Lately, I'm more and more interested in sampling, and it'll be great if we can have a project around the `sample` (`sample.int`) function (at C level) and make sure that [unequal probability sampling works as expected](https://www.sciencedirect.com/science/article/pii/S1574013722000673).

**Reply by hturner** [reply]
@dickoa can you post your idea to the ideas channel so it’s easier to keep track? 🙏
https://github.com/r-devel/r-project-sprint-2023/discussions/categories/ideas

**Comment by s-u** [comment]
Hello everyone!

My name is Simon Urbanek (Šimon Urbánek at my birthplace), I studied Statistics in Augsburg, Germany (specializing in visualization and interactive graphics), spent most of my time at [AT&T Labs - Research](https://www.research.att.com/) in NJ/NYC, USA (high-performance distributed and cloud computing) and in 2020 joined the [Dept. of Statistics](https://stat.auckland.ac.nz) at the University of Auckland, New Zealand (thus moving from the birthplace of S to that of R ;)) to teach data science skills required in real world based on my experience in the industry.

In R Core I am the developer and maintainer of the macOS version of R (see also https://mac.R-project.org). I am part of [CRAN](https://cran.R-project.org), taking care of the macOS builds of both R and all packages, providing binaries to users.

I wrote several R packages, mostly concerning performance, language bindings or infrastructure (see [my GitHub](https://github.com/s-u/)). I am also interested in tools for collaborative data science, co-authoring [RCloud](https://rcloud.social).

I would love to see interested Mac enthusiasts since our Mac GUI is of age and could do with some rewrite/improvements, but I'm also happy to oversee other infrastructure projects.

**Comment by MichaelChirico** [comment]
Hi all!

I'm Michael Chirico, from Pennsylvania but living in San Francisco where I've been at Google since 2020. Before that I was at Grab in Singapore, and my PhD in Economics before that in Philadelphia (which is when I picked up R).

I have contributed a lot to [{data.table}](https://github.com/Rdatatable/data.table) and [{lintr}](https://github.com/r-lib/lintr) and authored [{geohashTools}](https://github.com/MichaelChirico/geohashTools) and [{potools}](https://github.com/MichaelChirico/potools) (as well as a [few](https://github.com/cran/funchir) [minor](https://github.com/MichaelChirico/vcov) [packages](https://github.com/MichaelChirico/kueh)), and made 100s of smaller PRs to dozens of other packages, as well as a decent number of patches to R itself.

I am particularly interested in translation/internationalization and translation workflows.

**Comment by AyushBipinPatel** [comment]
Hello Everyone,

I am Ayush Patel. I am based in India. I was trained as an electrical engineer and then studied economics. Since then I have been working at the intersection of Economics (what is usually referred to as development economics or political economy of development) , Data and Right's based laws. I am currently affiliated to [Oxford Poverty and Human Development Initiative](https://ophi.org.uk/) (OPHI), University of Oxford. Here I help OPHI with Global Multidimensional Poverty Index, maintaining a Data Bank and a [Web Application](https://trainingidn.shinyapps.io/OPHIDataBankGlobalComparison/).

My first encounter with R was in the year 2015. Since then R has been my primary instrument to apply statistical analyses, carry out exploratory and descriptive analyses, create visualizations (I do dip my toes in JS when I need some help with interactivity) and communicate results through reports or web applications.

Apart from this, I am a [Posit (RStudio) certified tidyverse instructor](https://education.rstudio.com/trainers/people/patel+ayush/). I enjoy teaching foundations of R for data analysis, data visualization and reproducible reporting.

For long I have used R as an instrument. Lately, this lane is feels narrow as I move towards developing packages. This pushes me to study R as a Subject and not just use it as an instrument. I am here to help in any small incremental steps where I can and am excited to learn from all participants.

P.S. Let me know if someone wants any snacks or sweets from India

**Comment by daroczig** [comment]
Hello Everyone,

I'm Gergely (for English-speaking folks: pronounced as "gergei") from Hungary. Previously I spent a few years in Los Angeles as well, but have been working remotely for fintech/adtech/healhtech/SaaS companies since 2016.

I've been using R for ~20 years, most actively in the past ten years, with some contributions to 

- some R packages (for reporting, e.g. [`pander`](http://rapporter.github.io/pander/); using R in prod, e.g. [`logger`](https://daroczig.github.io/logger/); API integrations, e.g. [`fbRads`](https://github.com/daroczig/fbRads), [`botor`](https://daroczig.github.io/botor/), [`binancer`](https://daroczig.github.io/binancer/))
- the local R community (running the [Budapest Users of R Network](https://www.meetup.com/budapest-users-of-r-network/) with 1700 members, organized the [first satRday conference](https://budapest.satrdays.org/) in 2016, then the [2nd eRum conf](https://2018.erum.io/) in 2018)
- teaching R/data transformations/visualizations/scheduling batch jobs etc at various Hungarian universities, most recently as a lecturer at CEU since 2016

What I really enjoy is integrating R in other tools, e.g. to drive web applications through APIs.

I have a personal homepage at [daroczig.github.io](https://daroczig.github.io) if anyone is interested in more details about my background.

I've started and I'm maintaining the Weblate server for R at [translate.rx.studio](https://translate.rx.studio), and planning to work on this service at the R Project Sprint 2023 as well -- will open a related idea soon (and later a proposal as well).

Looking forward to meeting you all in a few months!

**Comment by pmur002** [comment]
Hi
I'm Paul Murrell.  
Home for me has been the University of Auckland since 1987, first as a student, but nowadays I get paid to be there.
I am part of R-core because my PhD supervisor (Ross Ihaka) kept nagging me to do some R graphics stuff and I finally gave in.  
I also like to write unpopular R packages and then [write reports about them](https://www.stat.auckland.ac.nz/~paul/index.html).
Looking forward to hopefully working with anyone who is interested in the graphics systems in R and making them better.

**Comment by tslumley** [comment]
Hi, 
I'm Thomas Lumley. I'm at the University of Auckland now, but when I started working on R in 1996 I was in Seattle at the University of Washington.  Nowadays I mostly work on packages, especially survey and now svylme.  In the past I did bug fixes and things related to regression modelling.

**Comment by ajrgodfrey** [comment]
Hello all,

I live in Palmerston North, New Zealand, about two hours' north of Wellington. I came here for a planned year in 1993, and never left Massey University. I started using R around 2006 once my S-Plus days came to an abrupt end.

Any notoriety I have in the R community is courtesy of being the public face of blind people using R. I'm certainly not the only blind person people should know using R though because R remains the best option for blind people to choose if they rely on screen reading software.  I'd like to think that my efforts, including the BrailleR package and various presentations at UseR conferences, have helped other blind people get into using R at university, and then beyond their educational requirements. My greatest reward is when a blind student chooses to take a second  or subsequent course in Statistics.

I believe I was the first totally blind person to successfully gain employment in a full-time role as a Lecturer in Statistics, and to the best of my knowledge, that number hasn't grown above two (0, 1, 2, 1, 2,  ... if we include Data Science). Aside from teaching and my supervision responsibilities, I provide statistical advice for researchers across campus, and if the clients want much of my help, they will be using R too.

I continue to carry out investigations into the tools that best suit blind students by making sure that all content I create can be read by me. The improvements in accessibility and shifts in the practice of teaching has led to vast improvements for many blind students today, and I'm pleased to see others leading the charge for greater accessibility and inclusive teaching and research practices. Sighted people don't all use R the same way, so it is important to me that blind people also have options too. This means I am always looking for ways to improve my own efficiency and effectiveness in order to expand the options for others. I am therefore a fairly early investigator of new tools, and sometimes this means I am an early adopter. I admit though that I drop other options pretty quickly if I don't think they are accessible (screen readers) or I can't see the return on my investment.

In recent years, my time has increasingly been called on to provide assistance to various New Zealand Government agencies in the drive for disaggregated data with respect to disability in general. As an established leader within the blindness community, I work with leaders of other Disabled Peoples' Organisations which means I am kept well on top of issues that affect other disabled people. In 2022, I attended NZ's examination before the United Nations Disability Committee in Geneva as head of the Independent Monitoring Mechanism; this is sort of like going to the Olympic Games for disability advocates.

This all means that I've been less able to support R Forwards in recent times than happened when I was first "invited" to join that effort. Life has always been full of such opportunities and I say "yes" a lot more often than not.

I look forward to attending the Sprint, and finding out what I can contribute.

Jonathan

**Comment by neerajdhanraj** [comment]
Dear All,

Originally from Nagpur, India, I am currently based in Abu Dhabi, UAE, where I work as a Senior Researcher in the Renewable and Sustainable Energy Research Center at Technology Innovation Institute (TII), UAE; and Assistant Professor at the Center for Quantitative Genetics and Genomics (QGG) at Aarhus University, Denmark.

I was educated and trained in India and Denmark in engineering, data science, renewable energy, and computer science. At the crossroads of these fields, my research interests lie within the general area of theoretical and applied data science, time series analysis view on industrial applications such as digital twins, artificial intelligence, bioinformatics, statistical software development, visualization techniques, and predictive analysis; and on exploring the scope of the automation of the machine learning and data science methodologies.

[Here](https://www.neerajbokde.in/software/), I provide brief documentation of some R packages that I have contributed to and published.

I am eagerly looking forward to meeting you all during the R Project Sprint 2023. I anticipate that our collaboration during this event will lead to fruitful discussions and productive outcomes. See you there!

Neeraj Dhanraj Bokde

**Comment by hanneoberman** [comment]
Hi all! I'm Hanne, a PhD candidate at Utrecht University, The Netherlands. My research focuses mostly on missing data, but I'm also involved in teaching all things `R`, data science and statistics, and I'm a fervent open science/FOSS advocate. As a core member of the [amices group](https://github.com/amices), I'm a developer for the imputation package `mice`, and I've have authored the accompanying visualization package `ggmice` (among other things). Take a look at [my website](https://hanneoberman.github.io/) to see more of my work. I'm looking forward to meeting you all!

**Comment by llrs** [comment]
Hi all, My name is Lluís Revilla Sancho, I work in the biomedical field, I have some pacakges in CRAN and Bioconductor related to it and some that are not.

Aside from packages I contribute to R with translations and participating in several working groups and communities, mostly the R repository working group and the R contributors working group, but I am too at the Bioconductor code of conduct committee and in the [BiocClassesWorkingGroup working group](https://github.com/Bioconductor/BiocClassesWorkingGroup/), [rOpenSpain](https://ropenspain.es/) and [rOpenSci](https://ropensci.org).

While I can't come to the sprint, I might participate remotely by providing feedback/ideas if I can.

**Comment by gbganalyst** [comment]
Hello,

My name is Ezekiel Adebayo Ogundepo, and I am a certified  [Posit (RStudio) tidyverse instructor](https://education.rstudio.com/trainers/people/ogundepo+ezekiel_adebayo/) from Nigeria. Currently, I hold the position of Data Analytics Manager at [54gene](www.54gene.com), a biotechnology company. I have a strong passion for utilizing R and have made significant contributions to the R community. Notably, I developed the [forstringr ](https://gbganalyst.r-universe.dev/forstringr) package, which simplifies the `stringr` library for individuals transitioning from Microsoft Excel. Additionally, I have contributed to the development of `ralger`, a web scraping package, and the [big book of R](https://www.bigbookofr.com/). Furthermore, I am an active member of [AfricaR](https://r4africa.org/) and serve as an R programming trainer for [Data Science Nigeria (DSN)](https://www.datasciencenigeria.org/).

**Comment by aitap** [comment]
Hello everyone,

My name is Ivan. I'm working on a STEM PhD, but somehow I always end up solving my problems through the use of programming. I like helping people on the R mailing lists, and I also maintain [a few][cmocean] CRAN [packages][depcache].

I have a weak preference for closing some of [my R tickets] (whether with `FIXED` or `WONTFIX` is up to the R core). I could also help debug crashes in compiled code and write new C patches. Another idea I have is enhancing _Writing R Extensions_ with some collected wisdom from `R-pkg-devel` and more precise API documentation, but cannot do this alone (such a project would need more wisdom collectors, a native English speaker and a member of R core to demarcate the boundaries of what ought to be considered an extension API).

[cmocean]: https://cran.r-project.org/package=cmocean
[depcache]: https://cran.r-project.org/package=depcache
[my R tickets]: https://bugs.r-project.org/buglist.cgi?no_redirect=0&quicksearch=reporter%3Akrylov.r00t%40gmail.com

**Comment by trangdata** [comment]
Hi everyone — My name is Trang Lê (pronounced like _les_ in French). Originally from Viet Nam, I'm currently a data scientist at Bristol Myers Squibb based in Cambridge, MA, USA. I author and maintain [openalexR](https://github.com/ropensci/openalexR) and [treeheatr](https://github.com/trangdata/treeheatr). Looking forward to meeting and learning from you all in couple months!

**Comment by clente** [comment]
Hello everyone,

My name is Caio (pronounced KAI-oh). I'm a computer scientist with an MSc from [IME-USP](https://www.ime.usp.br/en/dcc), a data scientist at [Terranova](http://trnv.com.br/), and a teacher at [Curso-R](https://www.curso-r.com/).

I created a few R packages that are up on CRAN ([chess](https://github.com/curso-r/chess), [stockfish](https://github.com/curso-r/stockfish), [droll](https://github.com/curso-r/droll), and [scryr](https://github.com/curso-r/scryr)), plus some internal packages for work. I also wrote [_Zen of R_](https://curso-r.github.io/zen-do-r/), over 50 posts for [Curso-R's blog](https://blog.curso-r.com/), and a bunch more stuff for [my own blog](https://lente.dev/).

See you all at the Sprint!

**Comment by georgestagg** [comment]
Hi all! My name is George and I'm from the UK. I work at [Posit, PBC](https://posit.co) as an Open Source Software Engineer and I'm the lead developer on the [webR project](https://docs.r-wasm.org/webr/latest/).

My [website](https://gws.phd) contains information about my background and previous work, which was mostly in academia. I have a PhD in Mathematical Physics from the [University of Newcastle UK](https://www.ncl.ac.uk) and my research was in the field of quantum turbulence and vortex dynamics, primarily through the lens of large-scale HPC simulation.

Before working at Posit, I taught computational mathematics with R and Python at Newcastle University, and it was in improving my teaching materials that the webR project first originated.

I'm hoping to gain some experience working with the bug tracker and contributing directly to R core, something I've not done before, by submitting and/or picking up some existing bugs to fix. I also have a passing interest in graphics programming, so will keep a close eye on the proposed Quartz alpha mask proposal.

Looking forward to seeing you all there at the sprint.

**Comment by rmhirota** [comment]
Hi everyone! 

My name is Renata and I am a Master's student in Statistics at University of Sao Paulo, Brazil, and a co-organizer of [R-Ladies Sao Paulo](https://www.meetup.com/pt-BR/rladies-sao-paulo/).

I work as a researcher at the [Brazilian Jurimetrics Association](https://abj.org.br/) analyzing data from judicial courts. I also work as a freelancer in data journalism projects, mainly developing R packages and Shiny apps for [Núcleo Jornalismo](https://nucleo.jor.br/en/).

I'm looking forward to collaborating and learning from you all at the sprint!

**Comment by phillybroadbent** [comment]
Hi! I'm Philly, a Research Software Engineer at the University of Southampton in the UK. I used R throughout my PhD in Psychology and now teach R as part of my job.  I'm looking to get more involved with the R community and look forward to meeting and collaborating with you all at the Sprint!

**Comment by paocorrales** [comment]
Hi! I'm Pao corrales from Buenos Aires, Argentina. I started using R in 2014 for a small project related to meteorology and I've always found an excuse to continue using R, first for my degree dissertation and then for my PhD. Now I also teach R and related tools for a Data Science degree here in Argentina. I organize and teach [short workshops](https://paocorrales.github.io/courses/) whenever I can both in English and Spanish. 

I'm also part of many R communities, like R-Ladies and LatinR. I've collaborated on translation projects like the Teaching Tech Together book,  the Dev Guide of rOpenSci, and now R messages. 

I'm looking forward to the RSprint!

**Comment by reikookamoto** [comment]
Hello 👋 

My name is Reiko and I live in Ottawa, Canada 🇨🇦 I currently work at National Research Council Canada where I use R to perform exploratory data analysis and statistical modelling on behalf of other government departments and local start-ups. 

Prior to my current role, I worked at the British Columbia Ministry of Health out on the west coast. There, I helped the organization adopt open-source software through developing R packages ([`bcepirate`](https://github.com/bcgov/bcEpiRate)) and data pipelines to generate R Markdown reports as part of the COVID-19 response.

I'm always keen on taking part in R-related informal education opportunities whether that may be leading a workshop on behalf of an R-Ladies chapter or organizing a lunchtime coding workshop for my colleagues.

I hope to familiarize myself with the codebase and Bugzilla by first attending to small issues. As a stretch goal, it'd be cool to also work on some translations for Japanese!

Looking forward to meeting everyone!

**Comment by bnaras** [comment]
Hello, 
I am Balasubramanian Narasimhan and I beat you all in name length 😄 . Please call me "Naras" as all my friends do. I studied maths and computer science in India before obtaining my PhD in Statistics from Florida State University. I began using R almost as soon as it was open sourced. My current interests include machine learning, optimization, clinical trial methodology, and privacy-aware computation and several of my packages on CRAN reflect these interests. In 2016, I chaired the useR conference, a great moment for me as we hosted so many wonderful useRs on the Stanford Campus. Lately, I have been working to increase awareness and use of R in India by hosting several conferences there and providing financial support for students to participate. One of my goals in attending the R sprint is to think of ideas to increase the number of R developers in the Indian subcontinent. I look forward to talking to many of you at the sprint and over beers, of course.

**Comment by EllaKaye** [comment]
Hello 👋

I'm Ella and I live in Oxford, UK. I'm a Research Software Engineer in the Department of Statistics at the University of Warwick, where I work with @hturner on her fellowship "Sustainability and EDI (Equality, Diversity and Inclusion) in the R Project". This involves:

👩🏻‍💻 Contributing to the [R project](https://www.r-project.org/) through code, infrastructure & outreach. 
📦 Contributing to packages on the [Comprehensive R Archive Network (CRAN)](https://cran.r-project.org/).
🚀 Capacity-building in R and data science at Warwick University.
💪 Community-building in the R and RSE communities with a focus on EDI.

Being part of the organising committee for and a participant in the Sprint is part of this!

I first started working with R as part of a MSc in Statistics back in 2014. Since then I have used it in extensively in my research as a statistician for visualising, analysing and modelling data, as well as for personal projects. I particularly enjoy developing packages, some of which are [publicly available](https://ellakaye.co.uk/packages.html).

I love the R community! I am a co-founder and co-organiser of [rainbowR](https://rainbowr.netlify.app/), a community that supports, promotes and connects LGBTQ+ people who code in the R language. I am the [R Forwards](https://forwards.github.io/) community team co-lead. I am on the organising committee of the [Warwick R User Group](https://www.meetup.com/Warwick-useRs/) and was previously co-organiser of the [Oxford R User Group](https://www.meetup.com/Oxford-R-User-Group/). 

I like to share what I know/am learning about R with others through writing blog [posts](https://ellakaye.co.uk/posts.html) and speaking at [R-Ladies](https://rladies.org/) and [R User Group](https://www.meetup.com/pro/r-user-groups/) meetups. Here are some [talks](https://ellakaye.co.uk/talks.html) I’ve given. 

I've really enjoyed reading these introductions and I can't wait to meet you at the Sprint and welcome you to Warwick.

**Comment by eliocamp** [comment]
Hi, y'all 

I'm Elio, from Buenos Aires, Argentina. I'm on my last months of my PhD in atmospheric sciences. I've been using R for about 9 years or so for study and research. I've developed a few packages mostly related to my work and I also teach R in informal courses and also in university (right now I'm starting to teach a course on visualisation). 

I'm eager to meet y'all and participate on the sprint!

**Comment by mcnanton** [comment]
Hi all!
I'm María from Buenos Aires, Argentina. 
I studied Social Communication at the University of Buenos Aires and now I'm pursuing a Master's degree in Data Mining & Knowledge Discovery at the same university. I'm mostly interested in NLP, data project management, data projects in the public sector and data quality management.
I started using R about 5 years ago, and since then I've taught sort courses on R and used it in countless projects. I'm an active participant in different communities of practice centered around open science and R in Latin America (like RLadies Buenos Aires and LatinR), and I've also contributed to projects like Turing Way or The Carpentries. I currently lead a team in a statistics office in my city's Ministry of Health, where I'm focused on cultivating the adoption of open science and tools to solve many of the challenges we face as an organization of the public sector dealing with massive health data.
I'm thrilled to be a part of this sprint!

**Comment by gmbecker** [comment]
Hi All,

I'm Gabe Becker. I've collaborated extensively with multiple members of R-core on various improvements to R (~7 novel features and numerous bugfixes), most notably the ALTREP framework which I proposed and then collaborated on with Luke Tierney. I also worked alongside Martin Maechler to co-develop and co-host  the Contributing To R Workshop for UseR!2021. Finally, I worked with Heather as part of the 'selection commitee' for this event. Having read your applications, i'm excited to meet and work with all of you.

The above also means I'm an "old hat" at this type of thing, as it were, so if you have an questions please reach out.

**Comment by hturner** [comment]
Hi everyone - about time I introduce myself properly!

I'm currently working as a Research Software Engineer at Warwick University. I'm just over 2 years into a 5 year fellowship seeking to address sustainability and EDI (Equality, Diversity and Inclusion) issues in the R Project, working with @EllaKaye who has already introduced herself. Organizing this sprint at Warwick is part of that work. I trained as a statistician and have worked in academia as a researcher, in industry as a statistician (early drug discovery) and as a freelance statistician/R programmer. I have lead the development of several statistical modelling packages (gnm, BradleyTerry2, PlackettLuce) and made contributions to various other R packages and to fixing bugs in R (mostly lm/glm related bugs).

I will most likely work on the nlme bugs project at the sprint, but will also be on hand to help people find things to work on, make connections between people and generally help everyone make the most of the sprint.

**Comment by nzgwynn** [comment]
Hi everyone,


I'm gwynn and I'm really excited to be here and learn all about you. 

I first started learning `R` in 2002 but didn't really decide to learn it until 2011, when I started my PhD which I did with @tslumley at the University of Auckland. I also worked with Ross Ihaka and @pmur002 then. I've worked at Harvard Business School and Wharton Business School and will be starting a new position at the IRS soon. 

In terms of community, I organized RLadiesDC for 3 years, co-founded Harvard Data Science Initiative's RUG, and am the admin at RForwards, as well as the accessibility team lead. I'm interested in accessible data visualizations and accessibility generally. I co-founded a startup that works in this area with my husband.

Thanks for reading,

gwynn

**Comment by jeroen** [comment]
Hi all,

I am Jeroen, I live in Utrecht 🇳🇱. I have been using R since 2007 and I am part of the [rOpenSci](https://ropensci.org/about/) staff. 

Besides maintaining several [cran packages](https://jeroen.r-universe.dev/) I mostly work on [r-universe](https://r-universe.dev): a new platform by rOpenSci to help discovering and publishing R software and R based research material. 

On r-universe, users can easily search through all R packages from CRAN, Bioconductor, and many more self-published packages. The system collects extensive information on pkg contents (exports, manuals, vignettes, datasets, etc) and background (development activity, contributors, popularity, etc). All information is exposed via a nice website and public apis, and indexed by the search engine. 

For packages or package versions that are not (yet) on CRAN/Bioc, the system automatically builds binaries for mac/win/linux, such that users can install a package without needing a full build environment (just like cran).

My main interests for this sprint follow from the development of r-universe:

 - Efforts to further document and standardize the official CRAN build system, and develop tooling to let package authors easily test and reproduce CRAN results using public infrastructure. 
 - Propose to standardize the url format for Linux binary packages in a cranlike package repository, similar to MacOS binaries, for example: `/bin/linux/jammy-x86_64/contrib/4.3/`. Ideally generalize the format to be used for any os-distro-arch combination.
 - The webr wasm port by @georgestagg. This is very exciting technology. R-universe uses this on the server to [export package datasets](https://fosstodon.org/@jeroenooms/110299179903212170) on-demand to csv, xlsx, json, etc. We want to solve the hurdles so that we can start providing wasm binaries for all R packages on r-universe.
 - Generally collect ideas on how to make r-universe more useful, both for R users trying to find relevant work, and also for package authors to increase the visibility and impact of their work.

Moreover I can help other participants with anything related to package development, building R or packages on various platforms, or CI related things.

**Comment by deepayan** [comment]
Hi All,

I'm Deepayan Sarkar. I'm a statistician working at the Indian Statistical Institute, Delhi, India. 

I started using R as a graduate student in Madison, Wisconsin, when I also wrote the lattice package. I'm still interested in graphics, but nowadays I don't find much time to work on it, and use R mainly for teaching and research. My main contribution to R itself has been on utilities such as code tagging, completion, and more recently, the help system.

**Reply by eliocamp** [reply]
Since you've worked on the help system, you might be interested in [this discussion about internalisation of package documentation](https://github.com/r-devel/r-project-sprint-2023/discussions/19).

**Comment by bjungbogati** [comment]
Hello everyone, 

I'm [Binod Jung Bogati](https://www.linkedin.com/in/bjungbogati/) from Kathmandu, Nepal. By profession, I'm a statistical programmer (designation: Data Science - Associate Manager). At my company, we've been using R for the validation of clinical trial datasets. Also, we're building a web interface for Clinical Trial Reporting. By passion, I'm an organizer at R User Group Nepal. 

In R Project Sprint, I'm interested in contributing to translation activities. I'm hoping to learn more from everyone.

**Comment by SiyaoIsHiding** [comment]
Hello everyone,

I am Siyao (Jane) He, a Master of Software Engineering student from the University of California, Irvine. In this sprint, I will participate remotely and I will work on the low-level area (base, core, & C programming). 

Looking forward to working with you in the Sprint!

**Comment by shannonpileggi** [comment]
Hi all! 👋 I'm Shannon Pileggi (she/her). Previously I was a college professor of statistics, and more recently I began working at a contract research organization that specializes in prostate cancer clinical trials. I have one package on CRAN ([gtreg](https://shannonpileggi.github.io/gtreg/), for adverse event reporting in clinical trials). I am an occasional [blogger](https://www.pipinghotdata.com/) (though less frequent this year) and I serve on the R-Ladies Global Leadership Team. 💜 I also still very much enjoy teaching, and I frequently give R-Ladies workshops as well as various conference workshops.
